### PR TITLE
63: As a user I want to have SQL microservice

### PR DESCRIPTION
**Pull request by Zoranais**
AzureBlobStorageService, TextService and MongoService have been added/moved to the SqlService.
For AzureBlobStorageService and Text Service, endpoints were implemented.

### CODE REVIEW

**Review by MaxShelestiuk — DISMISSED** [review]
Seems like everything is ok

File: backend/Squirrel.Core/Squirrel.Core.DAL/Extensions/ServiceCollectionExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+namespace Squirrel.Core.DAL.Extensions;
+
+public static class ServiceCollectionExtensions
+{
+    public static void AddSquirrelCoreContext(this IServiceCollection services, IConfiguration configuration)
+    {
+        var connectionsString = configuration.GetConnectionString("SquirrelCoreDBConnection");
+        services.AddDbContext<SquirrelCoreContext>(options =>
+            options.UseSqlServer(
```
Comment (serhiiyasenev) [inline backend/Squirrel.Core/Squirrel.Core.DAL/Extensions/ServiceCollectionExtensions.cs:14]: what if we use `Porstgres`?

Comment (Zoranais) [reply backend/Squirrel.Core/Squirrel.Core.DAL/Extensions/ServiceCollectionExtensions.cs:14]: But we are using SQL Server in the project. Here, I just moved AddSquirrelCoreContext from Core.WebAPI/Extensions to Core.DAL to reuse it in SqlService.

Comment (serhiiyasenev) [reply backend/Squirrel.Core/Squirrel.Core.DAL/Extensions/ServiceCollectionExtensions.cs:14]: yeah, I think if we use 2 DB in `Console App` probably we should be able to select 2 DB here as well, but it seems you are right

**Review by nazar-kotiukha — APPROVED** [review]
It seems that everything has been added and used appropriately.

**Review by serhiiyasenev — APPROVED** [review]

**Review by OlegKarapysh — APPROVED** [review]

**Review by MaxShelestiuk — APPROVED** [review]

**Review by Anatolii-Ivanov — APPROVED** [review]

**Pull request merged**